cmd/restic/cmd_cat.go

```diff
@@ -165,7 +156,15 @@ func runCat(gopts GlobalOptions, args []string) error {
 
 		_, err = globalOptions.stdout.Write(buf)
 		return err
+	}
+
+	// load index, handle all the other types
```

Comment:
"All the other types" means blobs and typos. The two switches can be merged with LoadIndex in the blob case.

Reply:
In which case the comment "handle all types that don't need an index" above the first switch can also be removed.

Reply:
Is changed. Thanks